src/app/onion/old-dashboard/components/dashboard-item/dashboard-item.component.html

```diff
@@ -1,5 +1,5 @@
 <div *ngIf="!hidden" class="dashboard-item" [ngClass]="{'disabled': disabled, 'shrink': small}">
-  <div class="row-item">
+  <div class="row-item" (click)="actionPermissions('infoPanel')&&viewSidePanel.emit()" [ngClass]="[actionPermissions('infoPanel') ? 'clickable' : 'notClickable']">
```

Comment:
Out of curiosity, since I've been a little out of the loop on 508 things, why is there a click event on a div? Isn't that problematic?

Reply:
We have a new directive I have to change this when that PR gets merged. The directive will be used in place of the click event so that keyboard and mouseevents will make the function fire.

Reply:
Can we add a FIXME here that notes that?